Automattic/wp-calypso
# Update myhome homepage not home page

#### Changes proposed in this Pull Request

Addresses #53036 by updating checklist references in My Home to be 'Homepage' instead of Home Page

#### Testing instructions

Start a new site (so checklist is fresh/unchecked). Check the 'Edit Homepage' option with /home/.

(Note - I'm not sure what state/context is needed to see the 'Choose a theme' checklist item - but I have updated the tet in there too).

Related to #53036
